config/schema/graphql.schema.yml

```diff
@@ -12,7 +12,7 @@ graphql.graphql_servers.*:
       type: string
       label: 'Endpoint'
     debug:
```

Comment:
I think we should also rename this to "debug_flag" to be in line with the lib. Makes it also more clear that this is then an integer.

Then we have a new property in the schema, we should check if that is a problem then.